### PR TITLE
add translation library & some other languages

### PR DESCRIPTION
**Pull request by bnchdrff**
This PR moves @randymeech 's translations into a format usable by lingui, and also integrates lingui itself.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "test-watch": "NODE_ENV=test jest --watch",
     "start": "ROLLUP_WATCH=true rollup -c --watch",
     "build": "rollup -c",
+    "translate": "lingui extract --clean && lingui compile --typescript",
```
Comment (tylergaw) [inline package.json]: Can we add a note about this script in the ole readme?

**Comment by tylergaw** [comment]
I wonder why those conflicts are happening? That's weird. 

Will we still need to get #43 in or does this include those changes

File: package.json (hunk 1 above)
Comment (bnchdrff) [reply package.json]: good call

**Comment by bnchdrff** [comment]
this PR replaces #43

**Comment by bnchdrff** [comment]
i think `bundle.js` will always conflict because it's minified. this PR just removes categories.json so that is expected also.

**Pull request merged**